src/endpoint/endpoint.js

```diff
@@ -92,6 +92,9 @@ dbg.log0('endpoint: replacing old umask: ', old_umask.toString(8), 'with new uma
 /* eslint-disable max-statements */
 async function main(options = {}) {
     try {
+        // setting process title needed for letting GPFS to identify the noobaa endpoint processes see issue #8039.
+        process.title = 'noobaa';
```

Comment:
how about adding this as config with 'noobaa' as default value, so that it can qlso be turned off or modified if needed? something like -

```suggestion
        if (config.ENDPOINT_PROCESS_TITLE) {
            process.title = config.ENDPOINT_PROCESS_TITLE;
        }
```